rmk/src/storage/mod.rs

```diff
@@ -534,28 +567,46 @@ impl Value<'_> for StorageData {
                     if buffer.len() < 3 + TAP_DANCE_MAX_TAP * 4 {
                         return Err(SerializationError::InvalidData);
                     }
-                    let timeout = BigEndian::read_u16(&buffer[1..3]);
+                    let timeout_ms = BigEndian::read_u16(&buffer[1..3]);
```

Comment:
Can the storage of `TapDanceData` be merged into the `MorseKeyData` storage?

Reply:
I want to keep TapDance and Morse separate because of one important diference: (see: is_pattern_completed() )
The tap dance is able to make its decision immediately when a hold occures -> so you can have hold actions, like shift, alt, ctrl. While morse may do this only if you reach the longest pattern.

Due to the same reason, hrm decisions are important and valid for th, td, but not much for morse.

However for morse a separate gap and hold timeout config may be needed - the tests will show.